src/components/Hotel/HotelOptions/index.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+import HotelCard from "./HotelCard";
+
+export default function HotelOptions({ hotels }) {
+  return (
+    <Body>
+      {hotels.map((h) => (
+        <HotelCard key={h.id} hotelCard={h} />
+      ))}
+    </Body>
```

Comment:
Seria legal substituir esse `h` por `hotéis` pra deixar mais descritivo.
Esse comentário se aplica a situações semelhantes.

Reply:
gabriel, sobre isso to em duvida, pois o pedrão tinha falado que era bom usar siglas pequenas para esses maps, é melhor usar a sigla completa mesmo então?